test/functional/normalization/audit_logs_format_test.go

```diff
@@ -214,7 +214,7 @@ var _ = Describe("[Functional][LogForwarding][Normalization] message format test
 				//outputTestLog := logs[0]
 				//Expect(outputTestLog).To(FitLogFormatTemplate(outputLogTemplate))
 				results := strings.Join(raw, " ")
-				Expect(results).To(MatchRegexp("kind.*Event.*level.*debug"), "Message should contain the audit log: %v", raw)
+				Expect(results).To(MatchRegexp("kind.*Event.*openshift_audit_level.*Metadata"), "Message should contain the audit log: %v", raw)
```

Comment:
maybe need also add checking like:  
`Expect(results).To(MatchRegexp("kind.*Event.*level.*default"), "Message should contain the audit log: %v", raw)`
?

Reply:
ok

Reply:
@vparfonov updated the test